src/components/organisms/Dashboard/hooks.ts

```diff
@@ -120,8 +132,27 @@ export default (teamId?: string) => {
           : undefined,
       )
       .filter((project): project is Project => !!project);
-  }, [projectData?.projects.nodes]);
+  }, [projectNodes]);
+
+  const hasMoreProjects =
+    projectData?.projects.pageInfo?.hasNextPage || projectData?.projects.pageInfo?.hasPreviousPage;
+
+  const isRefetchingProjects = networkStatus === 3;
 
+  const getMoreProjects = useCallback(() => {
+    if (hasMoreProjects) {
+      fetchMore({
+        variables: {
+          after: projectData?.projects.pageInfo?.endCursor,
+          first: projectPerPage,
+        },
+        updateQuery: (prev, { fetchMoreResult }) => {
+          if (!fetchMoreResult) return prev;
+          return fetchMoreResult;
+        },
```

Comment:
Is this necessary? 
Can you explain what this is doing, and if it is fetching more and merging, aren't we already doing that in `src/gql/provider.tsx`?

Reply:
when I removed this , the system shows only the latest call's  results  without merging